### PR TITLE
[#5] Define new Gate type 'Any'

### PR DESCRIPTION
**Pull request by jcieslar**
It return same value which they receive

### CODE REVIEW

File: lib/gate/coercer.rb [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
     def initialize(engine, type, allow_nil: false)
-      unless Axiom::Types.const_defined?(type)
-        fail CoercionError, "Doesn't know how to coerce into #{type}"
-      end
-
       @engine = engine
       @type = type
       @allow_nil = allow_nil
-      @coercion_method = Axiom::Types.const_get(type).coercion_method
+      @coercion_method = set_coercion_method
```
Comment (jandudulski) [inline lib/gate/coercer.rb]: Actually you can remove this line and `:coercion_method` from `attr_readers` and just rename `set_coercion_method` to `coercion_method`.

File: test/gate/test_coercer.rb [hunk 2]
```diff
@@ -6,6 +6,13 @@ def test_coerce_string
       assert_equal "o Hai", coercer_for(:String).coerce("o Hai")
     end
 
+    def test_coerce_any
+      assert_equal "o Hai", coercer_for(:Any).coerce("o Hai")
+      assert_equal [1, 2, 3] ,coercer_for(:Any).coerce([1, 2, 3])
+      assert_equal false ,coercer_for(:Any).coerce(false)
+      assert_equal true ,coercer_for(:Any).coerce(true)
```
Comment (jandudulski) [inline test/gate/test_coercer.rb]: Please reformat into `something, coercer_for(...)` (space after coma, not before ;))

Comment (jcieslar) [inline test/gate/test_coercer.rb]: lol... :D of course.

**Comment by jandudulski** [comment]
Can you also add line or two to the readme? Thanks!

**Pull request merged**